pegasus/sites.v3/code.org/views/congrats_continue.haml

```diff
@@ -1,17 +1,27 @@
-#continue{:style=>"background-color:grey; color:white; overflow:hidden; padding:20px; text-align:center"}
-  
-  %p{:style=>"font-size: 18px; font-family: 'Gotham 5r', sans-serif;"}
-    %b=I18n.t :continue_learning_header
-
-  %a{:href=>'http://studio.code.org'}
-    %button.button-color-share{:style=>'margin:10px'}
-      %b=I18n.t :learn_next_level
-
-  %a{:href=>'/learn/beyond'}
-    %button.button-color-share{:style=>'margin:10px'}
-      %b=I18n.t :learn_online
+%link{href: "/css/congrats-continue.css", rel: "stylesheet"}
 
+%h2=I18n.t :continue_learning_header
+%div.image-collection
+  %a.white-link{:href=>'http://studio.code.org'}
+    %div.choice
+      %div.choice-container
+        %div.choice-label.purple=I18n.t :learn_next_level
+        %img.character-img{src: "/images/congrats_studio_esstudents.jpg"}
+  %a.white-link{:href=>'/learn/beyond'}
+    %div.choice
+      %div.choice-container
+        %div.choice-label.purple.tall=I18n.t :learn_online
+        %img.character-img{src: "/images/congrats-beyond-other.jpg"}
   -if request.site != 'uk.code.org'
-    %a{:href=>'/learn/local'}
-      %button.button-color-share{:style=>'margin:10px'}
-        %b=I18n.t :learn_classes
+    %a.white-link{:href=>'/learn/local'}
+      %div.choice
+        %div.choice-container
+          %div.choice-label.purple=I18n.t :learn_classes
+          %img.character-img{src: "/images/map_local_classes.png"}
+  - if script == 'mc'
+    %a.white-link{:href=>'/learn/beyond'}
+      %div.choice
+        %div.choice-container
+          %div.choice-label.green
+            Go back to minecraft
```

Comment:
No. Apps is "protected" and it will happen with deploy but this does not happen with pegasus. Happy to make the change if you like. Let me know when it's ok to push to your PR.